api/internal/git/repospec.go

```diff
@@ -46,9 +46,6 @@ type RepoSpec struct {
 	// Branch or tag reference.
 	Ref string
 
-	// e.g. .git or empty in case of _git is present
-	GitSuffix string
```

Comment:
super nit: When we parse url path segments (rather than file path segments): https://github.com/kubernetes-sigs/kustomize/pull/4985/files#diff-109bb418d3031aaee374127b42b05a6845a4fb528a637c734a84270be49d8c8fR191-R192,
should we use "/" instead of `filepath.Separator`, which can be `\`?

Reply:
This is a great catch! I will submit a follow-up with this fix.